vtysh/bgp_vty.c

```diff
@@ -3451,7 +3453,22 @@ cli_neighbor_set_peer_group_cmd_execute(char *vrf_name, const char *ip_addr,
         }
     }
 
-/* If peer group has a remote-as, it becomes primary. */
+    /* Synchronize peer timers with group timers. */
+    key_timers[0] = OVSDB_BGP_TIMER_KEEPALIVE;
+    key_timers[1] = OVSDB_BGP_TIMER_HOLDTIME;
+
+    if (!ovs_bgp_peer_group->n_timers) {
```

Comment:
@mfwlarionov 
ovs_bgp_peer_group->n_timers == 0 means, that default timers is used. If we have a new one-timer feature, this logic will give us an error. 
@AKochin 
Ok, I reuse BGP_MAX_TIMERS